### ISSUE TITLE
`InteroUses` should populate `loclist`

### ISSUE THREAD

**Issue by stellarhoof** [parsonsmatt/intero-neovim#124]
Right now, `InteroUses` behaves no differently than searching naively. If I wanted this, I'd search for the identifier myself. This adds a couple of problems:
- Noise in the buffer (`hlsearch` is set by default)
- False positives (even strings in comments are selected)

**Comment by owickstrom** [comment]
+1 on of this, but wouldn't it be better to populate quickfix, instead of loclist, as there might be usages in other files?

**Comment by stellarhoof** [comment]
Better yet, provide an option for it. If it hardcodes one it might conflict with other plugins (e.x. linters). And so far from my testing, it only return usages for the current file. Maybe I'm missing something?

Ah also provide an option for customizing auto-open behavior :)

**Comment by parsonsmatt** [comment]
Yeah, my implementation for `Uses` is definitely a hack :smiley: I'd be happy for a better implementation, but I'm not sure the best route forward there.
